pages/account/index.vue

```diff
@@ -25,6 +25,25 @@
           </template>
         </AccountSettingsItem>
 
+        <AccountSettingsItem
+          icon="i-material-symbols-card-giftcard-outline-rounded"
```

Comment:
seems this icon name is invalid
```suggestion
          icon="i-material-symbols-featured-seasonal-and-gifts-rounded"
```

<img width="483" height="294" alt="Image" src="https://github.com/user-attachments/assets/7b88a1f3-a02c-4d6d-9bb1-8a68a25e1deb" />

<img width="890" height="409" alt="Image" src="https://github.com/user-attachments/assets/8d5ae1d2-cfa6-4f2c-b891-dc6cbdbc08d9" />

<img width="626" height="469" alt="Image" src="https://github.com/user-attachments/assets/c3087e3f-adc0-49c0-bc2b-40f4f0f5893b" />